packages/containers/src/EditableText/EditableText.container.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EditableText as Component } from '@talend/react-components';
+import Immutable from 'immutable';
+import omit from 'lodash/omit';
+import { cmfConnect } from '@talend/react-cmf';
+
+export const DISPLAY_NAME = 'Container(EditableText)';
+export const DEFAULT_STATE = new Immutable.Map({
+	editMode: false,
+});
+
+class EditableText extends React.Component {
+	static displayName = DISPLAY_NAME;
+
+	static propTypes = {
+		...cmfConnect.propTypes,
+		actionCreatorCancel: PropTypes.string,
+		actionCreatorEdit: PropTypes.string,
+		actionCreatorSubmit: PropTypes.string,
+		actionCreatorChange: PropTypes.string,
+		actionCreatorGoBack: PropTypes.string,
```

Comment:
This is not used and is proper to SubHeader not Editable Text

Reply:
oops